src/MessageHandler.ts

```diff
@@ -62,61 +60,31 @@ export default class MessageHandler {
   }
 
   private handleSoundCommands(message: Discord.Message) {
-    const sounds = Util.getSounds();
     const voiceChannel = message.member.voiceChannel;
-
     if (!voiceChannel) {
       message.reply('Join a voice channel first!');
       return;
     }
 
+    const sounds = Util.getSounds();
     switch (message.content) {
       case 'leave':
       case 'stop':
         voiceChannel.leave();
```

Comment:
I might not be following the behavior here, although I see that behavior does not change compared to before this commit.
Some questions:

- Is it possible that the bot has joined multiple voice channels at once?
- If yes, and I tell it to stop, will it then clear the queue stop playing for all voice channels?

Reply:
One can only be in one VoiceChannel at the same time.